test/k8s-integration/main.go

```diff
@@ -333,6 +333,8 @@ func generateTestSkip(normalizedVersion string) string {
 		// bug-fix introduced in 1.17
 		// (https://github.com/kubernetes/kubernetes/pull/81163)
 		skipString = skipString + "|volumeMode\\sshould\\snot\\smount\\s/\\smap\\sunused\\svolumes\\sin\\sa\\spod"
+		// Skip Snapshot tests pre 1.17
+		skipString = skipString + "|External.*Storage.*snapshot"
```

Comment:
Do we need the "External Storage" part?

Reply:
Given that the integration tests initiated from this PD CSI repo is passed in as --storage.testdriver, we can simply skip snapshot tests using "snapshot"